libs/data-display/src/DataTable.tsx

```diff
@@ -126,6 +126,12 @@ export type DataTableProps<T extends object> = {
    * Content to be displayed when the dataset is empty.
    */
   emptyState?: React.ReactNode;
+
+  /**
+   * Enable or disable multi-column sorting.
+   * When set to true, users can sort by multiple columns simultaneously.
+   */
+  enableMultiSort?: boolean;
```

Comment:
can you rename this to multisort / multiSort? "enable" is a bit redundant for a boolean prop